src/core/libmaven/mzSample.cpp

```diff
@@ -337,6 +337,7 @@ void mzSample::parseMzMLChromatogramList(const xml_node& chromatogramList)
         string chromatogramId = chromatogram.attribute("id").value();
         int sampleNo = getSampleNoChromatogram(chromatogramId);
 
+        // why is this cleaning needed?
```

Comment:
sometimes the scan number or RT etc is added to the SRM ID. This makes every ID unique and hence, useless as an identifier for scans emerging from a single source.

cleanFilterLine just removes such extraneous data.

Reply:
Got it. Thanks. Will remove that comment.